lib/fastlane/actions/xcodebuild.rb

```diff
@@ -1,4 +1,3 @@
-# rubocop:disable all
```

Comment:
You can leave this line here, so that you can get rid of the `TimeZone` part of the config file